ext/standard/basic_functions.stub.php

```diff
@@ -1179,18 +1179,147 @@ function random_int(int $min, int $max): int {}
 
 function soundex(string $string): string|false {}
 
+/* type.c */
+
+/** @param mixed $var */
+function gettype($var): string {}
+
+function settype(&$var, string $type): bool {}
+
+/** @param mixed $value */
+function intval($value, int $base = 10): int {}
+
+/** @param mixed $value */
+function floatval($value): float {}
+
+/** @param mixed $value */
+function boolval($value): bool {}
+
+/** @param mixed $value */
+function strval($value): float {}
+
+/** @param mixed $value */
+function is_null($value): bool {}
+
+/** @param mixed $value */
+function is_resource($value): bool {}
+
+/** @param mixed $value */
+function is_bool($value): bool {}
+
+/** @param mixed $value */
+function is_int($value): bool {}
+
+/** @param mixed $value */
+function is_float($value): bool {}
+
+/** @param mixed $value */
+function is_numeric($value): bool {}
+
+/** @param mixed $value */
+function is_string($value): bool {}
+
+/** @param mixed $value */
+function is_array($value): bool {}
+
+/** @param mixed $value */
+function is_object($value): bool {}
+
+/** @param mixed $value */
+function is_scalar($value): bool {}
+
+/** @param mixed $value */
+function is_callable($value, bool $syntax_only = false, &$callable_name = null) {}
+
+/** @param mixed $value */
+function is_iterable($value): bool {}
+
+/** @param mixed $value */
+function is_countable($value): bool {}
+
 /* uniqid.c */
 
 #ifdef HAVE_GETTIMEOFDAY
 function uniqid(string $prefix = "", bool $more_entropy = false): string {}
 #endif
 
+/* url.c */
+
+function parse_url(string $url, int $component = -1): array|string|int|false {}
+
+function urlencode(string $string): string {}
+
+function urldecode(string $string): string {}
+
+function rawurlencode(string $string): string {}
+
+function rawurldecode(string $string): string {}
+
+/** @param resource $context */
+function get_headers(string $url, int $format = 0, $context = UNKNOWN): array|false {}
+
+/* user_filters.c */
+
+/** @param resource $brigade */
+function stream_bucket_make_writeable($brigade): ?object {}
+
+/** @param resource $brigade */
+function stream_bucket_prepend($brigade, object $bucket): void {}
```

Comment:
These two return false for "Object has no bucket property". Convert to ValueError?

Reply:
Done